src/server/appserver.cpp

```diff
@@ -269,6 +269,12 @@ AppServer::AppServer(int &argc, char **argv)
 #endif
     if (KDC::isVfsPluginAvailable(VirtualFileMode::Suffix, error)) LOG_INFO(_logger, "VFS suffix plugin is available");
 
+#ifdef Q_OS_MACOS
+    // Init Updater
+    std::function<void()> quitCallback = std::bind_front(&AppServer::sendQuit, this);
```

Comment:
Is it possible to reuse here the `QuitCallback` type you have created below?

Reply:
Done